velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -763,11 +945,20 @@ static bool compareOrderedPartitions(
       return false;
     }
   }
-  if (not compareMaterializedRows(left.second, right.second)) {
-    return false;
-  }
 
-  return true;
+  std::vector<MaterializedRow> leftReordered{
```

Comment:
It seems like tryBucketMaterializedRows can be called with empty xxxReordered. The function can fill these in on its own. This will make the calling code simpler.